_topic_map.yml

```diff
@@ -1239,6 +1239,8 @@ Topics:
     File: creating-machineset-gcp
   - Name: Creating a MachineSet on OpenStack
     File: creating-machineset-osp
+  - Name: Creating a MachineSet on RHV
```

Comment:
All instances of "MachineSet" in this PR should be "machine set". 

We lowercase when speaking of these terms generally (machine set). And then use the official object kind and backtick when referring to the actual object (`MachineSet`). See the guidelines here for more info: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

(Note that these other instances in the topic map are being fixed in #28715)

Reply:
also please differentiate using RHV with
 Distros: openshift-enterprise,openshift-webscale,openshift-dedicated,openshift-online
But oVirt with
 Distros: openshift-origin

Reply:
@bergerhoffer Fixed. @sandrobonazzola I'm not sure specifically where you're talking about. Do you just mean to use {rh-virtualization} instead of specifying "oVirt" or "RHV"? If that's what you mean, I've fixed it.